documentation/src/main/asciidoc/topics/proc_disabling_authentication.adoc

```diff
@@ -1,10 +1,11 @@
-[id='disable_endpoint_auth']
-= Disabling {brandname} Server Authentication
+[id='disabling-authentication_{context}']
+= Disabling Authentication
 In local development environments or on isolated networks you can configure
 {brandname} servers to allow unauthenticated client requests.
 
 .Procedure
 
+. Open `infinispan.xml` for editing.
```

Comment:
When disabling authentication, we should instruct the user to disable authorization as well.

Reply:
good point. fixed.